wandb/apis/importers/wandb.py

```
@@ -0,0 +1,1133 @@
+                    try:
+                        future.result()
+                    except Exception:
+                        continue
+
+    def use_artifact_sequences(
+        self,
+        sequences: Iterable[ArtifactSequence],
+        config: Optional[ImportConfig] = None,
+        max_workers: Optional[int] = None,
+    ) -> None:
+        with progress.live:
+            with ThreadPoolExecutor(max_workers) as exc:
+                futures = {
+                    exc.submit(self.use_artifact_sequence, seq, config): seq
+                    for seq in sequences
+                }
+                for future in progress.task_pbar.track(
+                    as_completed(futures),
+                    description="Use Artifact Sequences",
+                    total=len(futures),
+                ):
+                    try:
+                        future.result()
+                    except Exception:
+                        continue
+
+    def import_reports(
+        self,
+        reports: Iterable[Report],
+        config: Optional[ImportConfig] = None,
+        max_workers: Optional[int] = None,
+    ) -> None:
+        """Import a collection of wandb.Reports.
+
+        Use `config` to specify alternate settings like where the report should be uploaded
+
+        Optional:
+        - `max_workers` -- set number of worker threads
+        """
+        with progress.live:
+            with ThreadPoolExecutor(max_workers) as exc:
+                futures = {
+                    exc.submit(self.import_report, report, config): report
+                    for report in reports
+                }
+                for future in progress.task_pbar.track(
+                    as_completed(futures),
+                    description="Reports",
+                    total=len(futures),
+                ):
+                    try:
+                        future.result()
+                    except Exception:
+                        continue
```

Ignoring exceptions silently is not a good idea.